infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -310,6 +335,53 @@ private String getLocalEndpoint() {
     }
 
     /**
+     * Task to collect local node metrics.
+     * This task collects the following:
+     *  Boolean Status of layout, sequencer and logunit servers.
+     *  Logunit Metrics:
+     *      Size of writeQueue for pending write requests.
+     * These metrics are then composed into NodeMetrics protobuf model
+     * and stored locally.
+     */
+    private void localMetricsPollingScheduler() {
+        // Initializing the status of these nodes to false.
+        Boolean layoutServerStatus = false;
+        Boolean sequencerServerStatus = false;
+        Boolean logunitServerStatus = false;
+        // No need to poll unless node is bootstrapped.
+        if (latestLayout == null) {
+            return;
+        }
+        try {
+            // Get status of componenets.
+            layoutServerStatus = getCorfuRuntime().getRouter(getLocalEndpoint()).getClient(LayoutClient.class).ping().get();
```

Comment:
indentation should be corrected here. Also, it might help to declare a variable that holds getCorfuRuntime().getRouter(getLocalEndpoint()).